modules/ROOT/pages/index-self-managed.adoc

```diff
@@ -1,26 +1,76 @@
 = Runtime Fabric on Self-Managed Kubernetes
 
-Runtime Fabric on Self-Managed Kubernetes enables you to deploy Mule applications and API proxies to a Kubernetes cluster that you create, configure, and manage. Runtime Fabric runs as a service on an existing Amazon Elastic Kubernetes Service (Amazon EKS), Azure Kubernetes Service (AKS), or Google Kubernetes Engine (GKE) environment.
+Runtime Fabric on Self-Managed Kubernetes enables you to deploy Mule applications and API proxies to a Kubernetes cluster that you create, configure, and manage. Runtime Fabric runs as a service on an existing Amazon Elastic Kubernetes Service (Amazon EKS), Azure Kubernetes Service (AKS), or Google Kubernetes Engine (GKE) environments.
 
-== Architecture
+== Core Components of Runtime Fabric on Self-Managed Kubernetes 
 
-Runtime Fabric on Self-Managed Kubernetes and its related components run as services on your Amazon EKS, AKS, or GKE environment. The following diagram shows different layers of the architecture.
+Runtime Fabric is a set of components that run inside your Kubernetes cluster and a set of components that run in the Anypoint control plane.
 
-image::architecture-self-managed.png[]
+The control plane pulls all Runtime Fabric core components and Mule Runtime images from the Runtime Fabric Docker registry. All Runtime Fabric in-cluster components run on Linux-based nodes and are managed by the Runtime Fabric agent.
 
-MuleSoft is responsible for providing support for Runtime Fabric services, the Mule Runtime server, and Mule applications. You are responsible for creating, configuring and managing the Amazon EKS, AKS, or GKE environment.
+=== The Runtime Fabric Agent
 
-== Shared Responsibility
+The Runtime Fabric agent is deployed on all nodes and communicates with the control plane via an mTLS outbound connection established at startup. 
```

Comment:
The Runtime Fabric agent is deployed as a Pod in the cluster and communicates with the control plane via an mTLS outbound connection established at startup. (its not deployed on all nodes)

Reply:
got it. thanks.